src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
@SandroMachado I took the liberty to re-organize a bit the props part in the Readme.md file by showing them all in a table, ordered alphabetically. 

One thing I also updated in the documentation is that -according to the source code- the `size` prop supports `normal` but only `small` and `large` was documented.

If you don't want these changes, I can revert the commit and simply add a new bulletin with the `cancelable` prop.